### PR TITLE
Pin 1417

### PR DESCRIPTION
**Pull request by beetlecrunch**

### CODE REVIEW

**Review by ubaldop — APPROVED** [review]

File: party-management-client/project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=1.3.10
```
Comment (galales) [inline party-management-client/project/build.properties]: any reason for this old version?

File: project/Dependencies.scala [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import Versions._
+import sbt._
+
+object Dependencies {
+
+  private[this] object akka {
+    lazy val namespace  = "com.typesafe.akka"
+    lazy val stream     = namespace           %% "akka-stream"      % akkaVersion
+    lazy val http       = namespace           %% "akka-http"        % akkaHttpVersion
+    lazy val httpJson4s = "de.heikoseeberger" %% "akka-http-json4s" % "1.38.2"
```
Comment (galales) [inline project/Dependencies.scala]: may we move the version in the proper file?

File: project/ProjectSettings.scala [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    // lifts some useful data in BuildInfo instance
+    val buildInfoExtra: Seq[BuildInfoKey] = Seq[BuildInfoKey](
+      "ciBuildNumber"    -> sys.env.get("BUILD_NUMBER"),
+      "commitSha"        -> commitSha,
+      "currentBranch"    -> currentBranch,
+      "interfaceVersion" -> interfaceVersion
+    )
+
+    def enableContractTest: Project = {
```
Comment (galales) [inline project/ProjectSettings.scala]: is this required?

**Review by galales — APPROVED** [review]

File: party-management-client/project/build.properties (hunk 1 above)
Comment (beetlecrunch) [reply party-management-client/project/build.properties]: No. Just copied from another project.

File: project/ProjectSettings.scala (hunk 3 above)
Comment (beetlecrunch) [reply project/ProjectSettings.scala]: No, not required.

**Pull request merged**